dev-python/sphinx-prompt/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+    <email>python@gentoo.org</email>
```

Comment:
@gentoo/python should sign-off on new packages under their maintainership.

Also, I think the commit title should say something like 'dev-python/sphinx-prompt: Add new package'.

Reply:
I added this package because it is a dependency for building flake8's documentation:

```
# Sphinx and friends are not keyworded for all arches, repoman complains
#distutils_enable_sphinx docs/source dev-python/sphinx-prompt dev-python/sphinx_rtd_theme
```

While the package works, repoman is unhappy that sphinx and other deps do not have the same keywords as flake8. So, for flake8 I either have to drop arches, or leave out the doc support. (or restricting the usage of the doc flag to certain arches, but I tried that once and it gets messy)

I figured that even if I can't add doc support to flake8 at the moment, I could still add the sphinx-prompt package in case it becomes possible to add this doc support later. (in case sphinx gains some keywords)

As flake8 is maintained by the python team, I added them to sphinx-prompt metadata as well. As far as I know, the python team allows that they are added as maintainer to packages that are dependencies of packages they already maintain. Though it would be great if someone from the python team could confirm this.